titration/utils/ui_state/user_value/volume.py

```diff
@@ -2,7 +2,7 @@
 The file for the Volume class
 """
 
-from titration.utils.ui_state.user_value.user_value import UserValue
+from AlkalinityTitrator.titration.utils.ui_state.user_value.user_value import UserValue
```

Comment:
Why does this include the project folder now? The folder name is arbitrary. I think "pylint" is being run outside the project, instead of from the root of the project.

Reply:
Yes, that was the case, this issue has been fixed.